feature/user_profile/build.gradle.kts

```diff
@@ -0,0 +1,29 @@
+plugins {
+    alias(libs.plugins.mifos.android.feature)
+    alias(libs.plugins.mifos.android.library.compose)
+    alias(libs.plugins.kotlin.parcelize)
+}
+
+
+android {
+    namespace = "org.mifos.mobile.feature.third.party.user_profile"
+}
+
+dependencies {
+    implementation(projects.ui)
+    implementation(projects.core.common)
+    implementation(projects.core.model)
+    implementation(projects.core.data)
+    
+    implementation(libs.squareup.retrofit2) {
+        // exclude Retrofit’s OkHttp peer-dependency module and define your own module import
+        exclude(module = "okhttp")
+    }
+    implementation(libs.squareup.retrofit.adapter.rxjava)
+    implementation(libs.squareup.retrofit.converter.gson)
+    implementation(libs.squareup.okhttp)
+    implementation(libs.squareup.logging.interceptor)
```

Comment:
why have you included retrofit dependencies? It should have implemented in data layer

Reply:
for httpexception error